test/Generators.Tests/AzureFunctionsTests.cs

```diff
@@ -11,7 +11,7 @@ public class AzureFunctionsTests
     const string GeneratedClassName = "GeneratedDurableTaskExtensions";
     const string GeneratedFileName = $"{GeneratedClassName}.cs";
 
-    [Fact]
+    [Fact(Skip = "Durable Functions Extension out of date")]
```

Comment:
Let's open a GitHub issue to track re-enabling these.

Reply:
Done: #85